<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Michell Stuttgart Faria</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>569</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Pynocchio Comic Reader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/main_window.qrc">
    <normaloff>:/icons/hand made/1424554308_comment.png</normaloff>:/icons/hand made/1424554308_comment.png</iconset>
  </property>
  <property name="accessibleName">
   <string>Pyellow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>628</width>
     <height>349</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="Viewer" name="viewer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(5, 5, 5);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QWidget" name="scroll_area_widget_contents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>626</width>
           <height>347</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resource/main_window.qrc">:/icons/hand made/1424554308_comment.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_recent_files">
     <property name="title">
      <string>&amp;Recent files</string>
     </property>
     <property name="icon">
      <iconset resource="../resource/main_window.qrc">
       <normaloff>:/icons/elementary3-icon-theme/places/48/folder-recent.svg</normaloff>:/icons/elementary3-icon-theme/places/48/folder-recent.svg</iconset>
     </property>
    </widget>
    <addaction name="action_open"/>
    <addaction name="action_open_folder"/>
    <addaction name="menu_recent_files"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_fullscreen"/>
    <addaction name="separator"/>
    <addaction name="action_original_fit"/>
    <addaction name="action_vertical_adjust"/>
    <addaction name="action_horizontal_adjust"/>
    <addaction name="action_best_fit"/>
    <addaction name="separator"/>
    <addaction name="action_rotate_left"/>
    <addaction name="action_rotate_right"/>
   </widget>
   <widget class="QMenu" name="menu_navegation">
    <property name="title">
     <string>&amp;Navegation</string>
    </property>
    <addaction name="action_next_page"/>
    <addaction name="action_previous_page"/>
    <addaction name="separator"/>
    <addaction name="action_first_page"/>
    <addaction name="action_last_page"/>
    <addaction name="separator"/>
    <addaction name="action_go_to_page"/>
    <addaction name="separator"/>
    <addaction name="action_next_comic"/>
    <addaction name="action_previous_comic"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_about_qt"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="action_show_toolbar"/>
    <addaction name="action_show_statusbar"/>
    <addaction name="separator"/>
    <addaction name="action_preference_dialog"/>
   </widget>
   <widget class="QMenu" name="menu_Bookmarks">
    <property name="title">
     <string>&amp;Bookmarks</string>
    </property>
    <addaction name="action_add_bookmark"/>
    <addaction name="action_remove_bookmark"/>
    <addaction name="separator"/>
    <addaction name="action_bookmark_manager"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_navegation"/>
   <addaction name="menu_view"/>
   <addaction name="menu_Bookmarks"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="separator"/>
   <addaction name="action_previous_comic"/>
   <addaction name="action_first_page"/>
   <addaction name="action_previous_page"/>
   <addaction name="action_next_page"/>
   <addaction name="action_last_page"/>
   <addaction name="action_next_comic"/>
   <addaction name="separator"/>
   <addaction name="action_go_to_page"/>
   <addaction name="action_add_bookmark"/>
   <addaction name="separator"/>
   <addaction name="action_rotate_left"/>
   <addaction name="action_rotate_right"/>
   <addaction name="separator"/>
   <addaction name="action_fullscreen"/>
   <addaction name="separator"/>
   <addaction name="action_exit"/>
  </widget>
  <action name="action_open">
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementaryicons3.1/actions/128/address-book-new.svg</normaloff>:/icons/elementaryicons3.1/actions/128/address-book-new.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementaryicons3.1/actions/128/help-contents.svg</normaloff>:/icons/elementaryicons3.1/actions/128/help-contents.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;About Pynocchio</string>
   </property>
  </action>
  <action name="action_about_qt">
   <property name="text">
    <string>&amp;About Qt</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementary3-icon-theme/actions/48/exit.svg</normaloff>:/icons/elementary3-icon-theme/actions/48/exit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_next_page">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementary3-icon-theme/actions/48/forward.svg</normaloff>:/icons/elementary3-icon-theme/actions/48/forward.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Next page</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="action_previous_page">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementary3-icon-theme/actions/48/back.svg</normaloff>:/icons/elementary3-icon-theme/actions/48/back.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Previous page</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_first_page">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementary3-icon-theme/actions/48/gtk-goto-first-ltr.svg</normaloff>:/icons/elementary3-icon-theme/actions/48/gtk-goto-first-ltr.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;First page</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="action_last_page">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementary3-icon-theme/actions/48/finish.svg</normaloff>:/icons/elementary3-icon-theme/actions/48/finish.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Last page</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="action_rotate_left">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementary3-icon-theme/actions/48/object-rotate-left.svg</normaloff>:/icons/elementary3-icon-theme/actions/48/object-rotate-left.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Rotate left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="action_rotate_right">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementary3-icon-theme/actions/48/object-rotate-right.svg</normaloff>:/icons/elementary3-icon-theme/actions/48/object-rotate-right.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Rotate right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="action_horizontal_adjust">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementary3-icon-theme/actions/48/calibrate (copy).svg</normaloff>:/icons/elementary3-icon-theme/actions/48/calibrate (copy).svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Horizontal adjust</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_fullscreen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementary3-icon-theme/devices/64/display.svg</normaloff>:/icons/elementary3-icon-theme/devices/64/display.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="action_go_to_page">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementary3-icon-theme/actions/48/filefind.svg</normaloff>:/icons/elementary3-icon-theme/actions/48/filefind.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Go to page...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_original_fit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementary3-icon-theme/actions/48/zoom-restore.svg</normaloff>:/icons/elementary3-icon-theme/actions/48/zoom-restore.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Original fit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_show_toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show Toolbar</string>
   </property>
  </action>
  <action name="action_show_statusbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S&amp;how Statusbar</string>
   </property>
  </action>
  <action name="action_add_bookmark">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementaryicons3.1/actions/128/bookmark-new.svg</normaloff>:/icons/elementaryicons3.1/actions/128/bookmark-new.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Add bookmark</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_remove_bookmark">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementaryicons3.1/actions/128/remove.svg</normaloff>:/icons/elementaryicons3.1/actions/128/remove.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove bookmark</string>
   </property>
  </action>
  <action name="action_bookmark_manager">
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementary3-icon-theme/actions/48/document-properties.svg</normaloff>:/icons/elementary3-icon-theme/actions/48/document-properties.svg</iconset>
   </property>
   <property name="text">
    <string>Bookmark &amp;manager</string>
   </property>
  </action>
  <action name="action_open_folder">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/freeiconmaker_21.png</normaloff>:/icons/icons/freeiconmaker_21.png</iconset>
   </property>
   <property name="text">
    <string>Open &amp;Folder</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_next_comic">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementary3-icon-theme/actions/48/stock_media-next.svg</normaloff>:/icons/elementary3-icon-theme/actions/48/stock_media-next.svg</iconset>
   </property>
   <property name="text">
    <string>Next Comic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Right</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="action_previous_comic">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementary3-icon-theme/actions/48/stock_media-prev.svg</normaloff>:/icons/elementary3-icon-theme/actions/48/stock_media-prev.svg</iconset>
   </property>
   <property name="text">
    <string>Previous Comic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Left</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="action_preference_dialog">
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementary3-icon-theme/actions/48/document-properties.svg</normaloff>:/icons/elementary3-icon-theme/actions/48/document-properties.svg</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="action_vertical_adjust">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementary3-icon-theme/apps/128/display-capplet.svg</normaloff>:/icons/elementary3-icon-theme/apps/128/display-capplet.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Vertical adjust</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_best_fit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/main_window.qrc">
     <normaloff>:/icons/elementary3-icon-theme/actions/48/zoom-more.svg</normaloff>:/icons/elementary3-icon-theme/actions/48/zoom-more.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Whole page</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Viewer</class>
   <extends>QScrollArea</extends>
   <header location="global">viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/main_window.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
